CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Change Log
+
+All notables changes to this project are documented in this file.
+
+The format is inspired by [Keep a Changelog].
+
+[Keep a Changelog]: https://keepachangelog.com/en/1.0.0/
+
+<!-- markdownlint-disable no-duplicate-heading -->
+
+<!-- NOTE: towncrier will not alter content above the TOWNCRIER line below. -->
+
+<!-- TOWNCRIER -->
+
+## 1.6.0 (2023-04-20)
```

Comment:
Please, include the Change Logs from the [release 1.0](https://github.com/oasisprotocol/oasis-wallet-web/releases/tag/v1.0) onwards.